spring-javaformat/spring-javaformat-checkstyle/src/main/java/io/spring/javaformat/checkstyle/check/SpringJUnit5Check.java

```diff
@@ -75,14 +67,6 @@
 		BANNED_IMPORTS = Collections.unmodifiableSet(bannedImports);
 	}
 
-	private static void addAnnotation(Set<String> annotations, String annotation) {
```

Comment:
I think that removing this method and the usage of it will introduce a regression. The [javadoc for `AnnotationUtil.containsAnnotation(DetailAst, List<String>)`](https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/utils/AnnotationUtil.html#containsAnnotation-com.puppycrawl.tools.checkstyle.api.DetailAST-java.util.List-) says that it "accepts both simple and fully-qualified names". If a user has annotated a method with, for example, `@org.junit.Test` rather than importing `org.junit.test` and annotating with `@Test`, this change will cause the usage to be missed.

Reply:
This method accepts both simple and fully-qualified names, e.g. "Override" will match both java.lang.Override and Override.

Which means that if we are using only the non FQN then we receive a check for both, there is also a test for that:
JUnit5BadAnnotation

Reply:
> if we are using only the non FQN

I don't think we can't be certain that this is the case. One user could write `@org.junit.Test` while another could write `@Test` along with `import org.junit.Test`.

Reply:
What I meant is that I believe it catches both

Reply:
Sorry, I see what you mean now. I agree that just the simple names should be sufficient.

We try to keep issues and pull requests focused on a single change. To that end, would you mind splitting the corrections to JUnit5BadModifier and the simplifications above into separate pull requests?

Reply:
Sure, I have updated this one to be the refactor, I'll create a new one for the test